dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumRawEvent.kt

```diff
@@ -204,6 +204,10 @@ internal sealed class RumRawEvent {
         override val eventTime: Time = Time()
     ) : RumRawEvent()
 
+    internal data class SDKInit(
```

Comment:
just to follow naming convention and to align with other usages of `SDK` word in the codebase (at least in v2 branch)

```suggestion
    internal data class SdkInit(
```